zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Collection.java

```diff
@@ -317,4 +356,17 @@ private void requireEditPermission(String email) throws UnauthorizedException {
             throw new UnauthorizedException("You are not authorised to edit collections.");
         }
     }
+
+    /**
+     * ScheduleCanceller is an abstraction wrapper for functionality to cancel a scheduled publish.
+     */
+    public static interface ScheduleCanceller {
```

Comment:
Not sure why the interface has been added to the same file. We don't normally do this?

Reply:
This is the only place this interface is/will ever but used so I created it within this class instead of in its own file.